core/commons.ts

```diff
@@ -62,32 +62,6 @@ export function deburrMatch(
 	return count;
 }
 
-const real_setInterval = global.setInterval;
-const real_clearInterval = global.clearInterval;
-
-const interval_cache = [];
-export function setInterval(
-	callback: (...args: any[]) => void,
-	ms?: number,
-): NodeJS.Timer {
-	const this_one = real_setInterval(callback, ms).unref();
-	interval_cache.push(this_one);
-	return this_one;
-}
-
-export function clearInterval(i: string | number | NodeJS.Timeout): void {
-	real_clearInterval(i);
-	const index = interval_cache.indexOf(i);
-	if (index !== -1) interval_cache.splice(index, 1);
-}
-
-export function clearAllIntervals(): void {
-	for (const interval of interval_cache) {
-		clearInterval(interval);
-	}
-	interval_cache.length = 0;
-}
-
```

Comment:
Oops, this change is not super relevant to this PR, but okay, I guess it's in there now. The idea is that there's no point maintaining `interval_cache` if the only thing we do with it is clear all intervals when Toadua exits – the runtime will already do that for us.

Reply:
Not a big deal but thanks for the heads-up!